osgp/shared/osgp-ws-smartmetering/src/main/resources/schemas/sm-installation.xsd

```diff
@@ -167,124 +164,122 @@
       <xsd:element name="Supplier" minOccurs="0">
         <xsd:simpleType>
           <xsd:restriction base="xsd:string">
-            <xsd:maxLength value="50" />
+            <xsd:maxLength value="50"/>
           </xsd:restriction>
         </xsd:simpleType>
       </xsd:element>
 
       <xsd:element name="HLS3_active" type="xsd:boolean"
-        minOccurs="0" />
+                   minOccurs="0"/>
       <xsd:element name="HLS4_active" type="xsd:boolean"
-        minOccurs="0" />
+                   minOccurs="0"/>
       <xsd:element name="HLS5_active" type="xsd:boolean"
-        minOccurs="0" />
+                   minOccurs="0"/>
       <!-- no type restriction for unencrypted keys for now -->
       <xsd:element name="Master_key" type="xsd:hexBinary"
-        minOccurs="0" />
+                   minOccurs="0"/>
       <xsd:element name="Global_encryption_unicast_key"
-        type="xsd:hexBinary" minOccurs="0" />
+                   type="xsd:hexBinary" minOccurs="0"/>
       <xsd:element name="Authentication_key" type="xsd:hexBinary"
-        minOccurs="0" />
-      <xsd:element name="Delivery_date" type="xsd:dateTime"
-        minOccurs="1" maxOccurs="1" />
+                   minOccurs="0"/>
+      <xsd:element name="Delivery_date" type="xsd:dateTime"/>
 
       <xsd:element name="Mbus_identification_number"
-        minOccurs="0">
+                   minOccurs="0">
         <xsd:simpleType>
           <xsd:restriction base="xsd:unsignedInt">
-            <xsd:maxInclusive value="99999999" />
+            <xsd:maxInclusive value="99999999"/>
           </xsd:restriction>
         </xsd:simpleType>
       </xsd:element>
 
       <xsd:element name="Mbus_manufacturer_identification"
-        minOccurs="0">
+                   minOccurs="0">
         <xsd:simpleType>
           <xsd:restriction base="xsd:string">
-            <xsd:pattern value="[A-Z]{3}"></xsd:pattern>
+            <xsd:pattern value="[A-Z]{3}"/>
           </xsd:restriction>
         </xsd:simpleType>
       </xsd:element>
 
       <xsd:element name="Mbus_version" minOccurs="0"
-        type="xsd:unsignedByte" />
+                   type="xsd:unsignedByte"/>
 
       <xsd:element name="Mbus_device_type_identification"
-        minOccurs="0" type="xsd:unsignedByte" />
+                   minOccurs="0" type="xsd:unsignedByte"/>
 
       <xsd:element name="Mbus_default_key" type="xsd:hexBinary"
-        minOccurs="0" />
+                   minOccurs="0"/>
 
     </xsd:sequence>
   </xsd:complexType>
 
   <xsd:complexType name="DeviceModel">
     <xsd:sequence>
-      <xsd:element name="Manufacturer" minOccurs="1"
-        maxOccurs="1">
+      <xsd:element name="Manufacturer">
         <xsd:simpleType>
           <xsd:restriction base="xsd:string">
-            <xsd:maxLength value="4" />
+            <xsd:maxLength value="4"/>
           </xsd:restriction>
         </xsd:simpleType>
       </xsd:element>
-      <xsd:element name="ModelCode" minOccurs="0" maxOccurs="1">
```

Comment:
Has the minOccurs been removed intentionally? Default value for minOccurs = 1.

Reply:
heel goed gezien, dat is idd niet de bedoeling, fixed.